lib/facter/pulpcore_workers.rb

```diff
@@ -0,0 +1,11 @@
+Facter.add(:pulpcore_workers) do
+  setcode do
+    directory = '/etc/systemd/system/multi-user.target.wants'
+    extension = 'pulpcore-worker@*.service'
+    return unless File.exist?(directory)
+
+    Dir[File.join(directory, "*#{extension}")].map do |config|
+      File.basename(config, extension)
```

Comment:
I don't think this works since it's not an extension but a glob pattern:
```console
$ irb
irb(main):001:0> File.basename('pulpcore-worker@2.service', 'pulpcore-worker@*.service')
=> "pulpcore-worker@2.service"
irb(main):002:0> File.basename('pulpcore-worker@2.service', '.service')
=> "pulpcore-worker@2"
```